components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -221,4 +258,119 @@ Should you have a need to revert back to IAM v1, it is quick and easy to do:
 $ chef-automate iam reset-to-v1
 ```
 
-Note that this discards your IAM v2 policies and roles, and re-configures Chef Automate to use your v1 policies again.
+Note that this discards your IAM v2 policies, roles, and projects, and re-configures Chef Automate to use your v1 policies again.
+
+<!-- TODO move this next to the rest of the projects walkthrough -->
+## Assigning Projects to Ingested Resources
+
+While projects can be directly assigned to resources created within Automate, ingested resources, including Compliance reports, events, and nodes, are created outside of Automate and therefore must be tagged with projects a different way.
+
+To tag any ingested resource with a project, that project must have at least one rule. A project rule contains conditions that
+determine if an ingested resource should be tagged with the rule's associated project. Each condition contains an attribute, operator, and value. See the [API reference]({{< relref "iam-v2-api-reference.md#project-rules" >}}) for details on how to manage project rules.
+Take this rule for the project `project-devops` as an example:
+
+```json
+{
+  "id": "devops-rule",
+  "name": "devops rule",
+  "type": "NODE",
+  "project_id": "project-devops",
+  "conditions": [
+    {
+      "operator": "EQUALS",
+      "attribute": "CHEF_ORGANIZATION",
+      "values": [
+        "devops"
+      ]
+    }
+  ]
+}
+```
+
+Once this project rule has been created, it must be applied in order to take effect. Once the project rule is in effect, all ingested resources with a `CHEF_ORGANIZATION` field that has the value `devops` will be considered a part of the `project-devops` project.
+Only these resources will appear in Automate when the `project-devops` project has been selected in the global project filter.
+
+{{% warning %}}
+Compliance reports must be using audit cookbook 7.5+ in order to make use of all of the available project rule attributes. Older reports will only have environment and role available as attributes.
+{{% /warning %}}
+
+In this example you will add a project rule to a project you just created. You will apply this new project rule so that all matching ingested resources get tagged with the appropriate project. You will then grant a user access to only that project
+and log in to that user's account to see their filtered view.
+
+First, determine which ingested resources should belong to the project. In this example, we want to tag Compliance reports
+with the Chef Organization `devops`, Infrastructure nodes with the environment `dev`, and actions on Chef Server `devops.pizza` with the project `project-devops`.
+You may want to verify that those filters work as expected using the search filter bars on the Event Feed, Client Runs, and
+Reports pages.
+
+{{% info %}}
+It is important to note that Compliance and Infrastructure resources will not necessarily match under the same conditions.
```

Comment:
```suggestion
Compliance and Infrastructure resources will not necessarily match under the same conditions.
```

Reply:
also, is this really true? im confused about this, shouldnt we just highlight that the nodes are different, like im pretty sure they can still have 1 rule to govern them both.

Reply:
that's at least what happens in my dev environment 🤷‍♀so ya that might be too literal. definitely would appreciate pointers on the wording here from superteam! @vjeffrey @lancewf

Reply:
If a condition for example of Chef Organization equals `devops` does not match both compliance and client runs nodes then that is a bug.

Reply:
+1 on what lance said -- if you've seen this behaviour in your dev env @bcmdarroch , could you open a bug issue detailing repro steps?

Reply:
just did some testing, and I found that if both an infra and compliance node match a condition, they get the project. however, this only worked when i used `Environment`. When I made a condition with `Chef Organization`, the infra nodes didn't get put in the project. :/ So I'll make a bug card for that. Then rewrite these lines